1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -5,6 +5,9 @@ describe('password validator', () => {
         it('returns an object', () => {
             expect(typeof validatePassword('test')).toEqual('object');
         })
+        it('returns an object with "isValid" property of boolean type', () => {
```

Comment:
Ah I see your approach. You're going to lead these forward by first focusing on the return object structure.

Admittedly, you didn't have a chance to see the lessons just yet but yes, see if you can make your tests focus more on the WHAT instead of the HOW. Make them more focused on the examples, and abstract away the shape or the return object from your tests here.

You can program by Wishful Thinking and arrange act assert backwards to feel it out, but you don't want tests that explicitly document the shape of your implementation.

Reason being this: it could change.

When you SPECIFY your tests with details that are too implementation focused, they become brittle and prevent you from pushing the code forward, making improvements.

You don't want to box yourself in with technical (HOW) decisions this early.

Specify the WHAT (concrete, behavioral examples) with your test names. Leave the HOW encapsulated by your TEST code.